source/main/menus/win-linux.js

```diff
@@ -8,9 +8,12 @@ import { environment } from '../environment';
 import { NOTIFICATIONS } from '../../common/ipc/constants';
 import { showUiPartChannel } from '../ipc/control-ui-parts';
 import { generateSupportRequestLink } from '../../common/utils/reporting';
+import { getRtsFlags } from '../utils/rtsFlags';
 
 const id = 'menu';
-const { isWindows, isBlankScreenFixActive } = environment;
+const { isWindows, isBlankScreenFixActive, network } = environment;
+const rtsFlags = getRtsFlags(network);
+const rtsFlagsEnabled: boolean = !!rtsFlags && rtsFlags?.length > 0;
```

Comment:
Please consider following (as discussed):

```js
const rtsFlagsEnabled: boolean = !!rtsFlags?.length && rtsFlags.length > 0;
```

This way we're sure that we compare `number` with a `number`.